data/json/items/generic/bathroom_house.json

```diff
@@ -0,0 +1,225 @@
+[
+    {
+        "id": "base_toiletries",
+        "name": "generic grooming",
+        "type": "GENERIC",
+        "description": "generic item template",
+        "symbol": "+",
+        "color": "blue",
+        "price": 200,
+        "price_postapoc": 0,
+        "weight": 100,
+        "volume": "50 ml",
+        "material": "plastic",
+        "bashing": 1,
+        "cutting": 0,
+        "to_hit": -1
+    },
+    
+    {
+        "id": "soap_holder",
+        "name": "soap dish",
+        "name_plural": "soap_dishes",
+        "type": "GENERIC",
+        "description": "A shallow dish for holding a bar of soap. It has ridges to help drain water away from the dish. Not the most exciting of items.",
+        "copy_from": "base_toiletries"
+    },
+    
+    {
+        
+        "id": "razor_shaving",
+        "name": "shaving razor",
+        "type": "GENERIC",
+        "description": "A razor blade on a comfortable handle. Much easier to shave with than a loose razor.",
+        "copy_from": "base_toiletries",
+        "price": 800,
+        "price_postapoc": 1200,
+        "//": "the end of the world doesn't mean everyone is okay with having ZZ Top beards. disposable razors are a hot commodity. The razor also loses the Cutting 1 quality. Don't ask how we confirmed this.",
+        "weight": 110,
+        "material": [ "plastic", "steel" ],
+        "cutting": 2
+        
+    },
+    
+    {
+        "id": "toothbrush_plain",
+        "name": "toothbrush",
```

Comment:
Can you add a recipe for the sharpened toothbrush with this? 🎉

Reply:
I'd say that is beyond the scope of this pr but absolutely should be done

Reply:
that's part of another thing I wanted to do later actually